packages/vertica-nodejs/lib/query.js

```diff
@@ -250,8 +251,28 @@ class Query extends EventEmitter {
     connection.sendCopyFail('No source stream defined')
   }
 
+  handleVerifyFiles(msg, connection) {
+    connection.sendVerifiedFiles(msg)
+  }
+
+  handleLoadFile(msg, connection) {
+    connection.sendCopyDataStream(msg)
+  }
+
+  handleWriteFile(msg, connection) {
+    if (msg.fileName.length === 0) { //using returnrejected, fileContents is an array of row numbers, not a string
+      this._result._setRejectedRows(msg.fileContents)
+    } else { // future enhancement, move file IO to util
+      fs.writeFile(msg.fileName, msg.fileContents, (err) => {
+        if (err) {
+          console.error('Error writing to file:', err);
+        }
+      });
+    }
+  }
+
   // eslint-disable-next-line no-unused-vars
-  handleCopyData(msg, connection) {
+  handleCopyDone(msg, connection) {
```

Comment:
Yes handleEndOfBatchResponse is in client.js. This is handleCopyDoneResponse, I can rename for clarification. 

Both handleCopyDoneResponse and handleEndOfBatchResponse are no-ops for the time being because this PR only supports single file loading and doesn't support copy from local stdin.

Reply:
The internal function was named properly, now they both are.